Code/GraphMol/RGroupDecomposition/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 
-rdkit_library(RGroupDecomposition RGroupDecomp.cpp RGroupUtils.cpp RGroupCore.cpp
+rdkit_library(RGroupDecomposition RGroupDecomp.cpp RGroupDecompData.cpp RGroupData.cpp RGroupUtils.cpp RGroupCore.cpp
```

Comment:
I think RGroupDecompParams.h also needs to be in the `rdkit_headers()` section of the CMakelLists.txt file. It is needed in client code